.github/workflows/test.yml

```
@@ -38,6 +38,7 @@ jobs:
           - ruby: "2.7"
           - ruby: "3.0"
             coverage: "yes"
+          - ruby: "3.1"
```

Should we continue running coverage on the newest Ruby version?
```suggestion
          - ruby: "3.1"
            coverage: "yes"
```